### DISCUSSION TITLE
Forum- APIs Done

### DISCUSSION THREAD

**Discussion by kovurusaisachin** [kovurusaisachin/AllyGator#143] [Show and tell]
@prashant-kapri Forum page related APIs are done. You can integrate these API with Front-end functionalities!!

**Comment by prashant-kapri** [comment]
okay cool, I will let you know if there's anything require from my end

**Comment by prashant-kapri** [comment]
I'm modifying the deliverable of the forum api by adding two new attributes to the api structure.

**Reply by prashant-kapri** [reply]
@kovurusaisachin  can you update the database with the new attribute.

**Reply by noopurthanvi** [reply]
@kovurusaisachin @prashant-kapri I have updated the api a bit and added a new endpoint so that we can get the post by forum id
